components/mail/sendmail/Makefile

```diff
@@ -183,22 +184,20 @@ COMPONENT_TEST_ENV +=	PATH=$(GNUBIN):$(USRBINDIR)
 # the results, so we can use a common "all" master file.
 COMPONENT_TEST_MASTER = $(COMPONENT_TEST_RESULTS_DIR)/results-all.master
 
-COMPONENT_TEST_TRANSFORMS += \
-    '-e "s|$(SM_BUILD_SUB_DIR)|\\$$(SM_BUILD_SUB_DIR)|g" ' \
-    '-e "s|$(OS_VERSION)|\\$$(OS_VERSION)|g" ' \
-    '-e "s|$(ARCH)|\\$$(ARCH)|g" ' \
-    '-e "s|^$(CC).*$$|XXX_CC_XXX|g" ' \
-    '-e "s|m$(BITS)|m\\$$(BITS)|g" ' \
-    '-e "/^XXX_CC_XXX\\$$/d" '
+COMPONENT_TEST_TRANSFORMS += -n \
+    '-e "/tests /p" ' \
+    '-e "/PASS: /p" ' \
+    '-e "/===========/p" '
 
 # Auto-generated dependencies
 PERL_REQUIRED_PACKAGES += runtime/perl
+REQUIRED_PACKAGES += $(ICU_LIBRARY_PKG)
 REQUIRED_PACKAGES += SUNWcs
 REQUIRED_PACKAGES += database/berkeleydb-5
-REQUIRED_PACKAGES += library/icu
 REQUIRED_PACKAGES += library/openldap
-REQUIRED_PACKAGES += library/security/openssl-11
+REQUIRED_PACKAGES += library/security/openssl-31
 REQUIRED_PACKAGES += runtime/perl-534
+REQUIRED_PACKAGES += runtime/perl-536
 REQUIRED_PACKAGES += shell/ksh93
```

Comment:
Only perl-538 is missing from the perl requirements. Is the dependency detection broken for perl? I haven't found any direct perl invocations for both versions 5.34 and 5.36 that are listed here. But perl-5.38 is our default now and not needed?

Reply:
I believe it is impossible to get these `REQUIRED_PACKAGES` using the automatic detection via `gmake REQUIRED_PACKAGES`, or something must be badly broken either in sendmail, or in the build framework.  @fritzkink, please run `gmake REQUIRED_PACKAGES` to update the list.